### PR TITLE
Feat: Rename RemoveCertificate to RemoveCredentials

### PR DESCRIPTION
**Pull request by FirestarJes**

### CODE REVIEW

File: source/marketparticipant/Energinet.DataHub.MarketParticipant.Application/Handlers/Actor/RemoveActorCredentialsHandler.cs [hunk 1]
```diff
@@ -51,15 +55,20 @@ public async Task Handle(RemoveActorCertificateCommand request, CancellationToke
...
         // Check that the actor has the correct type of credentials
-        if (actor.Credentials is not ActorCertificateCredentials credentials)
+        switch (actor.Credentials)
         {
-            throw new InvalidOperationException($"Actor with id {request.ActorId} does not have a certificate type credentials assigned");
+            case ActorCertificateCredentials certificateCredentials:
+                await _certificateService.RemoveCertificateAsync(certificateCredentials.KeyVaultSecretIdentifier).ConfigureAwait(false);
+                break;
+            case ActorClientSecretCredentials:
+                if (actor.ExternalActorId is not null)
```
Comment (defectiveAi) [inline source/marketparticipant/Energinet.DataHub.MarketParticipant.Application/Handlers/Actor/RemoveActorCredentialsHandler.cs:64]: Throw validation exception here.

File: source/marketparticipant/Energinet.DataHub.MarketParticipant.Application/Handlers/Actor/ActorRequestSecretHandler.cs [hunk 2]
```diff
@@ -54,31 +54,19 @@ public async Task<ActorRequestSecretResponse> Handle(ActorRequestSecretCommand r
...
-                case ActorClientSecretCredentials clientSecretCredentials:
-                    await _activeDirectoryB2CService
-                        .RemoveSecretsForAppRegistrationAsync(actor.ExternalActorId)
-                        .ConfigureAwait(false);
-                    break;
-                default:
-                    throw new InvalidOperationException("Unknown credentials type");
-            }
-
             actor.Credentials = null;
```
Comment (defectiveAi) [inline source/marketparticipant/Energinet.DataHub.MarketParticipant.Application/Handlers/Actor/ActorRequestSecretHandler.cs:63]: Remove this assignment.

**Review by xnetsrak — APPROVED** [review]

File: source/marketparticipant/Energinet.DataHub.MarketParticipant.Application/Handlers/Actor/ActorRequestSecretHandler.cs (hunk 2 above)
Comment (xnetsrak) [reply source/marketparticipant/Energinet.DataHub.MarketParticipant.Application/Handlers/Actor/ActorRequestSecretHandler.cs:63]: code block is removed, right ?

File: source/marketparticipant/Energinet.DataHub.MarketParticipant.Application/Handlers/Actor/RemoveActorCredentialsHandler.cs [hunk 3]
```diff
@@ -51,15 +56,22 @@ public async Task Handle(RemoveActorCertificateCommand request, CancellationToke
...
-        // Remove certificate from key vault
-        await _certificateService.RemoveCertificateAsync(credentials.KeyVaultSecretIdentifier).ConfigureAwait(false);
+                await _b2CService.RemoveSecretsForAppRegistrationAsync(actor.ExternalActorId).ConfigureAwait(false);
+                break;
+            default:
+                throw new InvalidOperationException($"Actor with id {request.ActorId} does not have a known type of credentials assigned");
+        }
 
-        // Remove certificate credentials from actor
+        // Remove credentials from Actor
```
Comment (defectiveAi) [inline source/marketparticipant/Energinet.DataHub.MarketParticipant.Application/Handlers/Actor/RemoveActorCredentialsHandler.cs:74]: Remove the comments, they just repeat the code.

**Review by defectiveAi — APPROVED** [review]

**Pull request merged**